learn-regular-expressions-by-building-a-spam-filter/spamFilter040.js

```diff
@@ -0,0 +1,16 @@
+const isSpam = msg => {
+  const re = /viagra|free money|work from home|stock alert|dear friend/;
+  /*
+  Looking good!
+
+  However, the spammers are getting smarter. They've learned to sometimes mix up the case of their key phrases. As well as "free money", our filter also needs to be able to handle "FREE MONEY", "free MONEY", or even "fReE MoNEy".
```

Comment:
This can be simplified:

```
The spammers are getting smarter -- on top of "free money", our regex also needs to be able to handle "FREE MONEY", "free MONEY", or even "fReE MoNEy". Add the ignore case flag `i` to the end of `re` to deal with those variations. For example, to create a case-insensitive regex for "foo bar", you would simply write `/foo bar/i`.
```